app/constants/memberships.js

```diff
@@ -104,6 +113,17 @@ export const allowedHasRelationWithMemberships = [
       ...AssistanceZoneCodeList,
     ],
   },
+  { organizations: [...DistrictCodeList], members: [...ProvinceCodeList] },
+  {
+    organizations: [...ProvinceCodeList],
+    members: [
```

Comment:
Hmm, good point it seems I inverted the direction of the membership between districts and provinces, as seen in line 116. If we want to keep a direct relation between these two it is probably be best to model them in the same direction as similar relations, thus indeed add district here.
I say "if" because such a direct relation did not exist before the introduction of memberships, back than a district was a sub organization of a municipality, which in turn was a sub organization of province. I'll check with Sofie, via the ticket, whether we want to introduce this additional direct relation or not.